tests/test_models.rs

```diff
@@ -1223,3 +1223,26 @@ fn test_null_id_in_tracklink() {
     assert!(linked_from.id.is_none());
     assert_eq!(linked_from.r#type, Type::Track);
 }
+
+#[test]
+#[wasm_bindgen_test]
+fn test_null_item_in_page() {
+    let json = r#"
+{
+  "href": "https://api.spotify.com/v1/search?offset=17&limit=2&query=My%20Playlist&type=playlist&market=FR",
+  "items": [
+    null
+  ],
+  "limit": 20,
+  "next": null,
+  "offset": 0,
+  "previous": null,
+  "total": 1
+}"#;
+    let page: Page<SimplifiedPlaylist> = deserialize(json);
+    assert_eq!(page.items.len(), 0);
+    assert_eq!(page.total-1, 0); // Because one element is null so ignore at deserialization but counted in total
```

Comment:
We could just assert the `page.total` equal `1` to indicate we don't change the `total` field at all.